application_manager/plugins/chronos/plugin.py

```diff
@@ -77,9 +74,10 @@ def start_application(self, data):
     def modify_job(self, api_rest_ip, jobname, job):
         updateCommand_1 = "startedAt=$(date +%s); "
         updateCommand_2 = "; /usr/bin/curl -H 'Content-type: application/json' -X POST " \
```

Comment:
please use 
update_command_2 = ("something" +
                                      "something2" +
                                      "something3")

it is better than \